weatherAplication/Modules/Home/View/HomeViewController.swift

```diff
@@ -2,83 +2,39 @@ import SnapKit
 import UIKit
 
 class HomeViewController: UIViewController {
-    private let kLogoWidth: CGFloat = 250
-    private let kLogoHeight: CGFloat = 250
-    private let kSideMargin: CGFloat = 10
-    private let kTopMargin: CGFloat = 20
-    private let kBottomMargin: CGFloat = 15
-    private let aplicationImage = R.image.weather_symbol()
-    private let logoImageView = UIImageView()
-    private let aplicationNameLabel = UILabel()
-    private let proceedButton = UIButton()
-    private let aplicationNameString: String = "MyWeather"
-    private let buttonString: String = R.string.localizable.button_string()
-
     private var viewModel: HomeViewModelContract
+    private var homeView: HomeView
 
     init() {
         viewModel = HomeViewModel()
+        homeView = HomeView(viewModelContract: viewModel)
         super.init(nibName: nil, bundle: nil)
     }
 
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
 
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        viewModel.viewWillAppear()
+    }
+
     override func viewDidLoad() {
         super.viewDidLoad()
-        setupView()
-        setupConstraints()
         viewModel.delegate = self
-        proceedButton.addTarget(self, action: #selector(buttonPressed), for: .touchUpInside)
+        view.addSubview(homeView)
+        homeView.snp.makeConstraints { make in
+            make.top.left.right.bottom.equalToSuperview()
+        }
```

Comment:
Nie robimy czegoś takiego, tylko zamiast tego w funkcji loadView robimy view = SomeView().

Reply:
dokladniej, to uzywamy czegos takiego
```
override func loadView() {
   view = MediaModuleView()
}
```
https://developer.apple.com/documentation/uikit/uiviewcontroller/1621454-loadview